contrib/terraform-testing-infrastructure/README.md

```diff
@@ -356,8 +356,14 @@ doing the following and then restarting Accumulo:
 ```bash
 cd ${software_root}/sources/accumulo-repo
 git pull
-mvn -s ${software_root}/apache-maven/settings.xml clean package -DskipTests -DskipITs
+mvn clean package -DskipTests -DskipITs
+# Backup the Accumulo configs
+mkdir -p ~/backup
+cp ${software_root}/accumulo/accumulo-${accumulo_version}/conf/* ~/backup/.
```

Comment:
"backup" is such a generic name. What if this directory already exists and contains other things? I know these are just example instructions, but users are likely just going to copy/paste these. It may be useful to select a more targeted/unique name in these instructions.

Reply:
Addressed in bbcb89e